homeassistant/components/devolo_home_network/device_tracker.py

```diff
@@ -83,14 +81,15 @@ def restore_entities() -> None:
     )
 
 
-class DevoloScannerEntity(
-    CoordinatorEntity[DataUpdateCoordinator[list[ConnectedStationInfo]]], ScannerEntity
+class DevoloScannerEntity(  # pylint: disable=hass-enforce-class-module
+    CoordinatorEntity[DevoloDataUpdateCoordinator[list[ConnectedStationInfo]]],
+    ScannerEntity,
```

Comment:
`ScannerEntity` should take precedence over `CoordinatorEntity`, and so pylint disable shouldn't be needed.
Is this branch up-to-date with latest code?

```suggestion
class DevoloScannerEntity(
    CoordinatorEntity[DevoloDataUpdateCoordinator[list[ConnectedStationInfo]]],
    ScannerEntity,
```

Reply:
Obviously not ... I did a rebase and now it is working 👍

Reply:
Done: https://github.com/home-assistant/core/pull/129429